### ISSUE TITLE
'Get Records' transactions not showing up in list

### ISSUE THREAD

**Issue by robert-at-hedera** [hashgraph/hedera-wallet-ios#176]
Reproduction steps:
1. Set up an account in the wallet with a few HBAR at minimum
2. From the side menu, select "Get Records"
3. Repeat step 2
4. Refresh the account
Expected result: The first "Get Records" transaction shows up in the overview page's transaction list.  This is what occurs on Android.
Actual result: No "Get Records" transactions show up, despite decreasing the account balance each time.

**Comment by avichalsaxena** [comment]
Can you test on the prod version? I am able to see the Get Records transaction in iOS as well.

**Comment by robert-at-hedera** [comment]
I can give it a try on my device again when I get home - sadly I left it there today, along with my mainnet recovery information.

**Comment by robert-at-hedera** [comment]
Alright, I've finished my research and proven to myself that this was my mistake.  Closing as not a bug; explanation below.

The short version is that I was testing iOS on mainnet with an account created by the Hedera Portal website.  When I switched to my other test account, everything behaved "as expected".

Sree had hinted to me that there was likely an issue due to the send / receive receipt thresholds, and that was exactly the issue.  The threshold for accounts created by the phone is "0", which I confirmed in the services code is not a special value.  The threshold for portal accounts must be higher.
